src/components/StatusBadge/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import Badge, { StyledBadge } from "./Badge";
+import Tooltip from "../Tooltip";
+import { themes } from "../../theme";
+
+import { badgeTypes, badgeProps } from "./constants";
+import { directions } from "../Tooltip/constants";
+
+const StatusBadgeCotainer = styled.div`
+  display: flex;
+  .badge {
+    margin-left: 8px;
+
+    &:first-child,
+    &.hidden {
+      margin-left: 0px;
+    }
+
+    &.hidden {
+      margin-top: 8px;
+      :first-child {
+        margin-top: 0;
+      }
+    }
+  }
+`;
+
+const HiddenBadgesCount = styled(StyledBadge)`
+  background: ${themes.global.transparent};
+  color: ${themes.global.onyx.light};
+`;
+
+const BadgeTooltip = styled(Tooltip)`
+  padding: 8px;
+`;
+
+class StatusBadge extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpened: false
```

Comment:
Hmm, now noticing we have some components using `this.state.isOpen` and some using `this.state.isOpened`. Let's move toward using `this.state.isOpen` for all new components.